generate_code_snippets.py

```diff
@@ -51,4 +51,7 @@ def collect_apis(obj, done):
             if openapi_ident in duplicate_operations:
                 snippets["operations"][duplicate_operations[openapi_ident]] += code
 
-print(json.dumps(snippets, indent=4))
+filename = "python-sdk-examples.json"
```

Comment:
I would recommend to run the script with `python3 generate_code_snippets.py > python-sdk-examples.json` instead. More flexible and less code

Reply:
fixed